src/pyop/userinfo.py

```diff
@@ -18,16 +18,22 @@ def __getitem__(self, item):
     def __contains__(self, item):
         return item in self._db
 
-    def get_claims_for(self, user_id, requested_claims):
+    def __delitem__(self, key):
+        del self._db[key]
```

Comment:
_note_: if https://github.com/IdentityPython/pyop/pull/41/files?diff=unified&w=1#r852268477 is removed, this should also be removed.

Reply:
agreed, removed.